js/app/modules/layout/topMenu.js

```diff
@@ -14,7 +14,7 @@ const DIRECTION = {
 };
 
 const DISTANCE_UP = 20;
-const DISTANCE_DOWN = 200;
+const DISTANCE_DOWN = 275;
```

Comment:
The code says 275, but the commit message 270. Either seems fine, but they should agree.

Reply:
Will check with @katelk28 , but for now I will assume the code is correct.